### DISCUSSION TITLE
CLI: `export` should default to temporary file or directory

### DISCUSSION THREAD

**Discussion by shykes** [dagger/dagger#12386] [Ideas]
When calling `export` from the CLI, the argument `path` is mandatory. This makes sense (you need to tell the engine where to export to), but the CLI should allow the argument to be omitted. When it is omitted, a temporary file (for `File.export`) or directory (for `Directory.export`) should be used.

**Comment by helderco** [comment]
Will need to hardcode setting it as optional because it's a required argument.
